map_gen/maps/crash_site/commands.lua

```diff
@@ -90,22 +90,42 @@ function Public.control(config)
         elseif state == 1 then
             local time_string = Core.format_time(game.ticks_played)
             Server.to_discord_named_raw(map_promotion_channel, crash_site_role_mention
-                .. ' **Crash Site has just restarted! Previous map lasted: ' .. time_string .. '!**')
+               .. ' **Crash Site has just restarted! Previous map lasted: ' .. time_string .. '!**')
 
             local end_epoch = Server.get_current_time()
             if end_epoch == nil then
                 end_epoch = -1 -- end_epoch is nil if the restart command is used locally rather than on the server
             end
 
+            local player_data = {}
+            for _, p in pairs(game.players) do
+                player_data[p.index] = {
+                    name = p.name,
+                    total_kills = ScoreTracker.get_for_player(p.index, 'player-total-kills'),
+                    spawners_killed =  ScoreTracker.get_for_player(p.index, 'player-spawners-killed'),
+                    worms_killed = ScoreTracker.get_for_player(p.index, 'player-worms-killed'),
+                    units_killed = ScoreTracker.get_for_player(p.index, 'player-units-killed'),
+                    turrets_killed = ScoreTracker.get_for_player(p.index, 'player-turrets-killed'),
+                    distance_walked = ScoreTracker.get_for_player(p.index, 'player-distance-walked'),
+                    player_deaths = ScoreTracker.get_for_player(p.index, 'player-deaths'),
+                    coins_earned = ScoreTracker.get_for_player(p.index, 'coins-earned'),
+                    entities_built = ScoreTracker.get_for_player(p.index, 'built-by-players'),
+                    time_played = p.online_time
+                }
+            end
+
             local statistics = {
+                scenario = data.scenario_name,
```

Comment:
```suggestion
                scenario = config.scenario_name,
```

`data.scenario_name` is the scenario that is going to be started. Could be different if an admin used the restart command.